packages/unity-bootstrap-theme/src/scss/extends/_form-fields.scss

```diff
@@ -366,6 +366,12 @@ form.uds-form {
   &.uds-form-gray7 {
     background-color: $uds-color-base-gray-7;
 
+    label, legend {
+      svg.uds-field-required {
+        color: $uds-color-font-dark-error;
```

Comment:
Please double check my comment and the XD. This should be gold.
`$uds-color-base-gold: #ffc627; // ASU Gold`

Let's correct that.

Also, thanks for cleaning up the BS4 references!